src/web/testing/figma-api-mocks.ts

```diff
@@ -36,49 +36,51 @@ export const mockFigmaMeEndpoint = ({
 	nock(baseUrl).get('/v1/me').reply(status, response).persist();
 };
 
-export const mockFigmaGetFileEndpoint = ({
+export const mockFigmaGetFileMetaEndpoint = ({
 	baseUrl,
 	fileKey,
 	accessToken,
-	query = {},
 	status = HttpStatusCode.Ok,
-	response = generateGetFileResponse(),
+	response = generateGetFileMetaResponse(),
 }: {
 	baseUrl: string;
 	fileKey: string;
 	accessToken: string;
 	query?: Record<string, string>;
 	status?: HttpStatusCode;
-	response?: GetFileResponse;
+	response?: GetFileMetaResponse;
 }) => {
 	nock(baseUrl, {
 		reqheaders: {
 			Authorization: `Bearer ${accessToken}`,
 		},
 	})
-		.get(`/v1/files/${fileKey}`)
-		.query(query)
+		.get(`/v1/files/${fileKey}/meta`)
 		.reply(status, response);
 };
 
-export const mockFigmaGetFileWithNodesEndpoint = ({
+export const mockFigmaGetFileEndpoint = ({
```

Comment:
**note:** Remove more specific `mockFigmaGetFileWithNodesEndpoint` in favor of more generic `mockFigmaGetFileEndpoint` used in the majority of tests.